test/module/irohad/ametsuchi/block_query_test.cpp

```diff
@@ -91,6 +91,15 @@ class BlockQueryTest : public AmetsuchiTest {
     }
   }
 
+  void TearDown() override {
+    transaction->abort();
+
+    postgres_connection->deactivate();
+    postgres_connection->disconnect();
```

Comment:
Multiple disconnects?

Reply:
What? There are `deACTIVATE` and then `disCONNECT`

Reply:
What? There are `deACTIVATE` and then `disCONNECT`

Reply:
I mean this pair and `TearDown` call from `AmetsuchiTest` below

Reply:
These are two different connections.